src/stream/src/executor/aggregation/agg_call.rs

```diff
@@ -60,4 +60,8 @@ pub struct AggCall {
     pub args: AggArgs,
     /// The return type of aggregation function.
     pub return_type: DataType,
+
+    /// Whether the stream is append-only.
+    /// Specific `StreamingAggStateImpl` may optimize its implementation based on this knowledge.
+    pub append_only: bool,
```

Comment:
> add a append_only flag to executor::aggregation::agg_call indicates that we can create an append-only agg state

It's also possible to eliminate this since all AggCalls in an agg executor must be with same `append_only`. This implementation also looks fine to me.